fullstack-cert/html-projects/framed-videos/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
it looks like there are some formatting issues here like in the iframe element itself.
I would suggest using something like prettier to auto format so you don't have to worry about it.